### PR TITLE
Adapt File API to modularized code base

### PR DESCRIPTION
**Pull request by felixvisee**
Jira issue: WP-1015

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   , "url" : "http://github.com/webinos/Webinos-Platform.git"
   }
 , "dependencies" :
-  { "find-dependencies" : "0.0.x"
+  { "webinos-jsonrpc2" : "*"
```
Comment (aldafu) [inline package.json:21]: please put this into devDependencies, that will be enough

**Comment by habibvirji** [comment]
can alex do the changes, then we can merge.. 

Regarding find-dependencies is it needed as the other code base does not require it to use...

**Comment by aldafu** [comment]
Felix' commits and the devDependency change.
https://github.com/webinos/webinos-api-file/pull/2

**Comment by felixvisee** [comment]
Sorry, was about to do that. So this one is no longer necessary.. closing.

**Pull request closed without merging**